concourse/tasks/delete_admin.yml

```diff
@@ -3,32 +3,46 @@ platform: linux
 image_resource:
   type: docker-image
   source:
-    repository: governmentpaas/cf-uaac
+    repository: governmentpaas/cf-cli
+    tag: feature_126965751_move_cf_cli_to_alpine
 inputs:
   - name: paas-cf
   - name: cf-secrets
   - name: cf-manifest
   - name: admin-creds
   - name: bosh-CA
+  - name: config
 run:
   path: sh
   args:
     - -e
+    - -u
     - -c
     - |
-      NAME=$(cat admin-creds/username)
-      if [ "${ENABLE_ADMIN_USER_CREATION}" == "false" ]; then
+      if [ "${ENABLE_ADMIN_USER_CREATION:-}" == "false" ]; then
         echo "Temporary user creation is disabled (ENABLE_ADMIN_USER_CREATION=${ENABLE_ADMIN_USER_CREATION}). Skipping."
         exit 0
       fi
       ./paas-cf/concourse/scripts/import_bosh_ca.sh
+      . ./config/config.sh
 
-      VAL_FROM_YAML=$(pwd)/paas-cf/concourse/scripts/val_from_yaml.rb
-      UAA_ADMIN_CLIENT_PASS=$($VAL_FROM_YAML secrets.uaa_admin_client_secret cf-secrets/cf-secrets.yml)
-      UAA_ENDPOINT=$($VAL_FROM_YAML properties.uaa.url cf-manifest/cf-manifest.yml)
+      NAME=$(cat admin-creds/username)
 
       echo "Removing user ${NAME}"
-      uaac target "${UAA_ENDPOINT}"
-      uaac token client get admin -s "${UAA_ADMIN_CLIENT_PASS}"
 
-      uaac user delete "${NAME}"
+      echo | cf login -a ${API_ENDPOINT} -u ${CF_ADMIN} -p ${CF_PASS}
+
+      cf delete-user "${NAME}" -f
+
+      echo "Removing leaked ${PREFIX} users"
```

Comment:
@richardc @keymon @mtekel and I discussed this after standup. Concluded that:

- we'll only use this script for a short amount of time; revert after it has reached production
  - we're confident enough that it will do the right thing for the short amount of time it'll be present
  - so we don't need additional guards or automated tests
  - but we'll test it again in dev against an environment that has a sample of "real" users
- we'll extend `sync-admin-users` to also use `cf delete-user` in another story, which will take care of leaked test users in future
- we might want to split the functionality of `sync-admin-users` to two separate tasks in future:
  - remove orphaned (test?) users, which would run in dev and make it clearer to team members that we have this functionality
  - create admin users which doesn't run in dev